### DISCUSSION TITLE
v14 Block grid UX request: edit action trigger area

### DISCUSSION THREAD

**Discussion by shearer3000** [umbraco/Umbraco-CMS#17002] [Features and ideas]
Currently, only the edit pencil icon acts as the button for the edit action (trigger to show the flyout/overlay)

Would it be possible for all/most of the visual component to act as the trigger for this action by default? i.e. as highlighted here:
![image](https://github.com/user-attachments/assets/0e29f57c-3939-4d30-a8d9-6d088034ac99)

This is so the content editor can click anywhere in the block to edit it (apart from the delete button, and without affecting drag and drop/move actions etc). 

This is similar to how the Nested content UI worked.

**Comment by shearer3000** [comment]
In addition to above, the block selector only allows you to choose a block by clicking the text but ideally the entire component would be the action trigger. Orange area of this example:
![image](https://github.com/user-attachments/assets/efa7f7a3-337e-42f3-826b-8ebe244445ed)

goal: make it that bit easier for content editor

**Comment by deanleigh** [comment]
> Currently, only the edit pencil icon acts as the button for the edit action (trigger to show the flyout/overlay)
> 
> Would it be possible for all/most of the visual component to act as the trigger for this action by default? i.e. as highlighted here: ![image](https://private-user-images.githubusercontent.com/2433680/364177011-0e29f57c-3939-4d30-a8d9-6d088034ac99.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzExNDIyNzYsIm5iZiI6MTczMTE0MTk3NiwicGF0aCI6Ii8yNDMzNjgwLzM2NDE3NzAxMS0wZTI5ZjU3Yy0zOTM5LTRkMzAtYThkOS02ZDA4ODAzNGFjOTkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEwOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMDlUMDg0NjE2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDhmYjhlMGFkZmIyZjE1OTM2MzNlZTdkM2M5NGJmNjEyYTFiNjQ5Y2UwN2Q5MzU5ZjhhMDZhMGUzOWU5NDM5MyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.l0cNPNg8xPWzI61lsXEx7X7MR26E4admGgxQIHpN6JI)
> 
> This is so the content editor can click anywhere in the block to edit it (apart from the delete button, and without affecting drag and drop/move actions etc).
> 
> This is similar to how the Nested content UI worked.

I was thinking similar but with a draggable part of the bar at the start usually consisting of two columns of three dots.
A fairly common ux pattern

**Reply by kasparboelkjeldsen** [reply]
@deanleigh it is also how blocklists works in v13, so it would make sense grid worked the same way.

**Comment by nielslyngsoe** [comment]
This should be working as expected now, and has done so for some major versions. Therefor I will close this discussion, if there is more to the topic that you like to have adressed please let me know and we can look into it.

Thanks in advance
